lib/connectors/memory.js

```diff
@@ -352,29 +352,61 @@ Memory.prototype.fromDb = function(model, data) {
   if (!data) return null;
   data = deserialize(data);
   var props = this._models[model].properties;
+
   for (var key in data) {
     var val = data[key];
-    if (val === undefined || val === null) {
-      continue;
-    }
+
     if (props[key]) {
```

Comment:
Line 370 is `if (val === undefined || val === null) {`, I am referring to a different check - `if (props[key]) {`